pages/common/gh-gist.md

```diff
@@ -0,0 +1,24 @@
+# gh gist
+
+> Work with GitHub Gists on the command line.
+> More information: <https://cli.github.com/manual/gh_gist>.
+
+- Create a new Gist from a space-separated list of files:
+
+`gh gist create {{path/to/files}}`
+
+- Create a new Gist with a description:
+
+`gh gist create {{filename}} --desc "{{description}}"`
+
+- Edit a Gist:
+
+`gh gist edit {{id_or_url}}`
+
+- List Gists you own:
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:

What about this?
```suggestion
- List Gists owned by the currently logged in user:
```